nipype/interfaces/ants/preprocess.py

```diff
@@ -0,0 +1,333 @@
+        return None
+
+    def _list_outputs(self):
+        outputs = self._outputs().get()
+        if _extant(self.inputs.output_spatial_map):
+            outputs['spatial_map'] = (
+                os.path.abspath(self.inputs.output_spatial_map)
+            )
+        if _extant(self.inputs.output):
+            outputs['output'] = os.path.abspath(self.inputs.output)
+        return outputs
+
+class AntsMotionCorrInputSpec(ANTSCommandInputSpec):
+    '''Input spec for the antsMotionCorr command.'''
+    dimension_desc = (
+        "This option forces the image to be treated as a "
+        "specified-dimensional image. If not specified, N4 tries to infer "
+        "the dimensionality from the input image."
+    )
+    dimensionality = traits.Enum(3, 2, argstr='-d %d', usedefault=True,
+                                 position=0, desc=dimension_desc)
+
+    average_image = File(argstr='-a %s', position=1, exists=False,
+                         desc="4D image to take an average of.")
+
+    output_average_image = traits.File(desc="Filename to save average of input image as.",
+                                       genfile=True, hash_files=False, argstr='%s')
+
+    output_transform_prefix = Str(
+        desc="string to prepend to file containg the transformation parameters"
+    )
+    output_warped_image = Str(desc="Name to save motion corrected image as.")
+
+    metric_type_desc = (
+        "GC : global correlation, CC: ANTS neighborhood cross correlation, "
+        "MI: Mutual information, and Demons: Thirion's Demons "
+        "(modified mean-squares). Note that the metricWeight is currently not "
+        "used. Rather, it is a temporary place holder until multivariate "
+        "metrics are available for a single stage."
+    )
+
+    metric_type = traits.Enum("CC", "MeanSquares", "Demons", "GC", "MI",
+                              "Mattes", argstr="%s", desc=metric_type_desc)
+    fixed_image = File(requires=['metric_type'],
+                       desc="Fixed image to do motion correction with respect to.")
+    moving_image = File(requires=['metric_type'],
+                        desc="This is the 4d image to be motion corrected")
+    metric_weight = traits.Float(1.0, requires=['metric_type'])
+    radius_or_bins = traits.Int(desc="", requires=['metric_type'])
+    sampling_strategy = traits.Enum("None", "Regular", "Random", None,
+                                    requires=['metric_type'])
+    sampling_percentage = traits.Either(traits.Range(low=0.0, high=1.0), None,
+                                        requires=['metric_type'])
+
+    transformation_model = traits.Enum("Affine", "Rigid", argstr="%s")
+    gradient_step_length = traits.Float(requires=['transformation_model'],
+                                        desc='')
+
+    iterations = traits.Int(
+        argstr="-i %d",
+        desc="Specify the number of iterations at each level."
+    )
+    smoothing_sigmas = traits.Int(
```

Comment:
as per command usage, smoothing_sigmas should accept the format MxNx0. This can be achieved with the ```sep``` meta argument:
```
smoothing_sigmas = traits.List(traits.Float, argstr='-s %f', sep='x', desc='...')
```

Please add a doctest exercising these multiple arguments

Please, also check that the command takes floats for the smoothing_sigmas or if it is in voxels (then ints).